apps/cookbook/src/app/examples/button-example/button-example.component.html

```diff
@@ -58,6 +58,22 @@ <h2>Button with icon only</h2>
   <label>Disabled</label>
 </div>
 
+<div class="button-example-container">
```

Comment:
If you read the cookbook, this makes very little sense, as all you see are two identical buttons w.o. any explanation: 
![Screenshot 2023-03-03 at 13 02 58](https://user-images.githubusercontent.com/114976818/222715631-d2b2ab72-1af6-4707-ab1d-ae2f4425be1b.png)

Reply:
Roger 😅 
I'll update the texts 👍🏻